examples/guru2d1_adjoint.cpp

```diff
@@ -25,35 +27,35 @@ int main(int argc, char *argv[]) {
   finufft_opts opts;
   finufft_default_opts(&opts);
   opts.upsampfac = 1.25;
-  complex<double> I(0.0, 1.0); // the imaginary unit
+  std::complex<double> I(0.0, 1.0); // the imaginary unit
 
   // generate random non-uniform points on (x,y) and complex strengths (c):
-  vector<double> x(M), y(M);
-  vector<complex<double>> c(M);
+  std::vector<double> x(M), y(M);
+  std::vector<std::complex<double>> c(M);
```

Comment:
the example is just unpleasant to look at now - why can't we using namespace std  for such example codes? They are not unit-tested.

Reply:
Ok, I'll get over it. Maybe ``using std::vector;`` etc would make everyone happy. I was just hoping the examples would be simple and pleasant to look at, but C++ is just not pleasant...